org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/web/WebIntegrationFragment.xtend

```diff
@@ -815,6 +815,8 @@ class WebIntegrationFragment extends AbstractXtextGeneratorFragment {
 				�ENDIF�
 				public class �grammar.servletClass.simpleName� extends �'org.eclipse.xtext.web.servlet.XtextServlet'.typeRef� {
 					
+					private static final long serialVersionUID = 1L;
```

Comment:
unfortunately this would give a warning in xtend

Reply:
Meh...